### DISCUSSION TITLE
CORS Policy in Vanjaro

### DISCUSSION THREAD

**Discussion by joshgk00** [vanjarosoftware/Vanjaro.Platform#916] [Q&A] [answered]
Is the CORS Policy different than that of a standard DNN install?

I'm trying to drop a Constant Contact Sign Up form on a Vanjaro site using the Custom Code block. I'm I'm getting an error about the CORS Policy

Access to XMLHttpRequest at '[RESOURCE_ON_CONSTANT_CONTACT_DOMAIN]' from origin '[MY_VANJARO_SITE]' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

I put the same code on a plain DNN site and it worked as expected, so I'm wondering if something may be different in relations to CORS Policy?

**Comment by meetmandeep** [comment, marked as answer]
No. There are no changes. Take a look at the web.config. Are you comparing the same version of DNN? It's possible that DNN changed the default web.config in the latest version.

**Reply by joshgk00** [reply]
I wasn't comparing the specific versions, but I will try that and see if I can spot any differences.

**Reply by joshgk00** [reply]
Turned out it was an issue on my end. I thought I had had the DNN and Vanjaro configured the same when I did the test, but upon a closer look, there was a slight difference on the code that I added, which causes the issue.
